wg-notebooks/charter.md

```diff
@@ -0,0 +1,39 @@
+# WG Notebooks Charter
+
+This charter adheres to the conventions, roles and organization management outlined in [wg-governance].
+
+## Scope
+
+WG Notebooks is responsible for the user experience around Notebooks and other IDEs and their integrations with Kubeflow.
+The WG covers launching and managing different types of Notebooks on Kubeflow as well as workflows that Notebook extensions enable.
+
+### In scope
+
+#### Code, Web apps, Controllers and Services
+
+- APIs and tools used for launching Notebooks (e.g. Notebooks CRD/Controller).
+- Web UI to manage and connect to Notebook instances (e.g. Notebook Manager UI)
+- SDKs and integrations from inside the Notebooks (e.g. Git inside Notebooks, Kale)
```

Comment:
Kale is not currently a Kubeflow project. So I don't think it should be part of the initial work group proposal. Given Kale is at the intersection of notebooks & pipelines; potentially either this WG or the KFP WG could be the right home for Kale at some point in the future.

That said, I think the priority right now within the community should be on finding WG sponsorship for existing projects and making sure WGs are running smoothly before adopting new projects.

So I think any discussion of moving Kale into Kubeflow should be postponed for the future.

Reply:
I am also missing something about maintaining the notebook images - i.e. keeping the deps up to date, making sure various kernels work in the environment etc.

Reply:
@jlewi ACK, I removed any Kale references